data/rte/snli/README.md

```diff
@@ -0,0 +1,49 @@
+The Stanford Natural Language Inference (SNLI) Corpus
+=====================================================
+
+
+More on the dataset at http://nlp.stanford.edu/pubs/snli_paper.pdf.
+
+Regarding software, use ``tools/snli_preprocess.py`` and ``tools/train.py``
+(on ``snli`` task).
+
+Model Comparison
+----------------
+
+For randomized models, 95% confidence intervals (t-distribution) will be reported.
+
+Note that NO DROPOUT is applied for any of the models.
+
+| Model                    | train    | test     | t. mean  | settings
```

Comment:
Awesome! Waiting for results with train+dev for training and eval'd on test too, then I think we are clear to merge!